Formula/kubie.rb

```diff
@@ -0,0 +1,43 @@
+class Kubie < Formula
+  desc "Much more powerful alternative to kubectx and kubens"
+  homepage "https://blog.sbstp.ca/introducing-kubie/"
+  url "https://github.com/sbstp/kubie/archive/v0.9.1.tar.gz"
+  sha256 "8f7926b1bcd0421247231353b6969c36498add3b88b55103349df3da763a5f42"
+  license "Zlib"
+
+  depends_on "rust" => :build
+  depends_on "kubernetes-cli" => :test
+
+  def install
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "."
```

Comment:
```suggestion
    system "cargo", "install", *std_cargo_args
```

Reply:
Thanks, I'll make that change. Is there documentation where I can find that kind of stuff myself, or alternatively is there a way to validate those things at build time and issue an error or a warning? Like a lint checker for `brew` Formula style / requirements.

Reply:
It's just a convenience function so I don't think we can really lint that. But generally it's looking at other formulae that will teach you about these.